TrustSDK/Classes/TrustConfiguration.swift

```diff
@@ -0,0 +1,18 @@
+//  Copyright © 2018 Trust.
+//
+//	This file is part of TrustSDK. The full TrustSDK copyright notice, including
+//	terms governing use, modification, and redistribution, is contained in the
+//	file LICENSE at the root of the source code distribution tree.
+	
+
+import Foundation
+
+public struct TrustConfiguration {
+    let scheme: String
+    let callbackPath: String
```

Comment:
supported coins?

Reply:
It's already implemented in `SigningOutputDecoder`